xml/System.Windows.Data/RelativeSource.xml

```diff
@@ -45,7 +45,7 @@
   
  :::code language="xaml" source="~/snippets/csharp/System.Windows/FrameworkElement/DefaultStyleKey/themes/generic.xaml" id="Snippetrelativesource":::  
   
- For the complete sample, see [NumericUpDown Custom Control with Theme and UI Automation Support Sample](https://go.microsoft.com/fwlink/?LinkID=160025).  
+ For the complete sample, see [NumericUpDown Custom Control with Theme and UI Automation Support Sample](https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp).  
```

Comment:
This one should be deleted. We shouldn't cross link into other repo snippet files, only samples.